.github/workflows/develop-ci-cd.yml

```
@@ -0,0 +1,147 @@
+        env:
+          COMMON_GITHUB_USER: ${{ secrets.COMMON_GITHUB_USER }}
+          COMMON_GITHUB_TOKEN: ${{ secrets.COMMON_GITHUB_TOKEN }}
+        run: |
+          cd ${{ matrix.service.folder }}
+          chmod +x gradlew
+          ./gradlew clean build -x test --no-daemon
+
+      # 6. AWS 자격증명 구성
+      - name: Configure AWS Credentials
+        if: ${{ contains(env.services_to_build, matrix.service.folder) }}
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)